masterlist.yaml

```
@@ -13486,6 +13486,30 @@ plugins:
         util: '[SSEEdit v4.0.3](https://www.nexusmods.com/skyrimspecialedition/mods/164)'
         nav: 2
 
+  - name: 'Knight of the North( - LCO Patch)?\.esp'
+    url:
+      - link: 'https://github.com/TateTaylorOH/Knight-of-the-North/'
+        name: 'Knight of the North - A Creation Club Quest Overhaul (GitHub)'
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/45869/'
+        name: 'Knight of the North - A Creation Club Quest Overhaul (Nexus Mods)'
```

The main location should go first:

```suggestion
      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/45869/'
        name: 'Knight of the North - A Creation Club Quest Overhaul (Nexus Mods)'
      - link: 'https://github.com/TateTaylorOH/Knight-of-the-North/'
        name: 'Knight of the North - A Creation Club Quest Overhaul (GitHub)'
```